src/panels/lovelace/components/notifications/hui-notification-item-template.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  css,
+  CSSResult,
+} from "lit-element";
+
+import "../../../../components/ha-card";
+
+@customElement("hui-notification-item-template")
+export class HuiNotificationItemTemplate extends LitElement {
+  protected render(): TemplateResult | void {
+    // @apply: https://github.com/Polymer/lit-element/issues/633#issuecomment-475983474
+    return html`
+      <style>
+        ha-card .header {
+          @apply --paper-font-headline;
```

Comment:
This won't work, and if it works, it's because there is still some Polymer left in the tree. This will stop working when we kill the last Polymer!  We should not support this.

Reply:
I must have misunderstood your comment on here: https://github.com/home-assistant/home-assistant-polymer/issues/2946. I'll grab the paper-styles